apps/bare-expo/ios/Podfile.lock

```diff
@@ -47,18 +47,18 @@ PODS:
     - ExpoModulesTestCore
   - EXNotifications (0.29.8):
     - ExpoModulesCore
-  - Expo (52.0.6):
+  - Expo (52.0.7):
```

Comment:
not sure where this is coming from? maybe pod install hasn't been run in a while?